tests/app/celery/test_scheduled_tasks.py

```diff
@@ -646,14 +646,14 @@ def test_alert_if_letter_notifications_still_sending(sample_letter_template, moc
     two_days_ago = datetime(2018, 1, 15, 13, 30)
     create_notification(template=sample_letter_template, status='sending', sent_at=two_days_ago)
 
-    mock_celery = mocker.patch("app.celery.scheduled_tasks.deskpro_client.create_ticket")
+    mock_create_ticket = mocker.patch("app.celery.scheduled_tasks.zendesk_client.create_ticket")
 
     raise_alert_if_letter_notifications_still_sending()
 
-    mock_celery.assert_called_once_with(
+    mock_create_ticket.assert_called_once_with(
         subject="[test] Letters still sending",
         message="There are 1 letters in the 'sending' state from Monday 15 January",
-        ticket_type='alert'
+        ticket_type='incident'
```

Comment:
It feels like the `zendesk_client.TYPE_INCIDENT` abstraction is leaking here. I think you could use the constant here as well.